includes/Services/PluginService.php

```diff
@@ -21,6 +22,28 @@
  * Class for providing plugin related services.
  */
 class PluginService {
+
+	/**
+	 * Gets the list of plugins based on different enabled Hiive Flags
+	 *
+	 * @return array
+	 */
+	public static function initialize_hiive_flag_plugins() {
+		$init_plugins_extended = array();
+		$hiive_flags           = Plugins::get_hiive_plugin_flags();
+
+		foreach ( $hiive_flags as $hiive_flag => $is_enabled ) {
+			// Check if it is an allowed flag and is enabled on the container
+			if ( true === $is_enabled && true === Config::get_site_capability( $hiive_flag ) ) {
+				// Check if there are plugins for the flag.
+				if ( isset( Plugins::$hiive_flag_plugin_list[ $hiive_flag ] ) ) {
```

Comment:
Can we check for a non-empty array instead of using `isset()`?

Reply:
Sure